student.rb

```diff
@@ -0,0 +1,19 @@
+require './person'
+
+class Student < Person
+  attr_accessor :classroom
+
+  # constructor method
+  def initialize(classroom)
```

Comment:
- [ ] Hello! 🌟 In order to ensure that the Student class's initialize method accepts age, name (with a default value of "unknown"), and parent_permission (with a default value of true), it's important to pass these parameters to both the Student class's initialize method and the super call within the Student class. By doing so, you're ensuring that the Student class inherits all the attributes from its parent class, Person. 🚀📚